pkg/cloud/converters/tags.go

```diff
@@ -31,6 +32,11 @@ import (
 	infrav1 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
 )
 
+const (
+	// AwsInternalTagPrefix is the prefix for AWS internal tags, which are reserved for internal AWS use.
+	AwsInternalTagPrefix = "aws:"
+)
+
```

Comment:
This probably doesn't need to be exported right?

Reply:
yes, using it here.